docs/DeveloperGuide.adoc

```diff
@@ -1,4 +1,4 @@
-= AddressBook Level 3 - Developer Guide
```

Comment:
Are you sure this is still an address book? (You haven't removed all `AddressBook` branding from the developer guide.)